app/models/vote_seeder.rb

```diff
@@ -0,0 +1,25 @@
+class VoteSeeder
+  def self.seed_votes
+    reviews = Review.all
+    reviews.each do |review|
+      VoteSeeder.fake_review_votes(review)
+    end
+  end
+
+  def self.fake_review_votes(review)
+    users = User.all
+    user_count = users.count
+
+    number_of_votes = rand(user_count)
+    up_vote_users = users.sample(number_of_votes)
+    up_vote_users.each do |user|
+      FactoryGirl.create(:vote, review: review, user: user)
+    end
+    other_users = users - up_vote_users
+    other_users_count = other_users.count
+    down_vote_users = other_users.sample((other_users_count / 2).floor)
+    down_vote_users.each do |user|
+      FactoryGirl.create(:vote, :down, review: review, user: user)
+    end
+  end
+end
```

Comment:
Very cool logic here! Only small note on convention. For this I would organize this into a seeders directory in your app directory instead of your models because this class is working a differently than your other models. Then the class methods and way you access them in the master seeds file is perfect!